packages/analytics/index.tsx

```diff
@@ -23,7 +23,7 @@ export default function usePageviewAnalytics({
 
   useEffect(() => {
     if (process.env.NODE_ENV === 'production') {
-      if (!siteId || !includedDomains) return
+      if (!siteId || !includedDomains || !navigator.sendBeacon) return
```

Comment:
I could be wrong but this looks like it might return early when perhaps the other `load()` call should indeed run below. 

I wonder if we should adjust our approach - maybe w/ an object to collect the features we care about, then warn based on that for each case. 

Just want to ensure we have the right logic + see if we can make it a bit more clear. That's my 2 cents.

Reply:
Oh, the `load()` call below shouldn't be run if `navigator.sendBeacon` doesn't exist. If the Fathom script does load without `navigator.sendBeacon` present, then the Fathom client calls later on will cause a 404 later on when trying to execute:

https://hashicorp.slack.com/archives/GATJMHBCJ/p1646069187301569?thread_ts=1646066937.856259&cid=GATJMHBCJ

Reply:
Perhaps it would be helpful to add a comment here for context on the behavior we're trying to avoid!